test/benchmarks/driver_bench/src/suites/multi_bench/grid_fs_download.mts

```diff
@@ -0,0 +1,40 @@
+import { Readable, Writable } from 'node:stream';
+import { pipeline } from 'node:stream/promises';
+
+import { driver, type mongodb } from '../../driver.mjs';
+
+export const taskSize = 52.43;
+
+let bucket: mongodb.GridFSBucket;
+let bin: Uint8Array;
+let _id: mongodb.ObjectId;
+const devNull = () => new Writable({ write: (_, __, callback) => callback() });
+
+export async function before() {
+  bin = await driver.load('single_and_multi_document/gridfs_large.bin', 'buffer');
+}
+
+export async function beforeEach() {
+  await driver.drop();
+  await driver.create();
+
+  bucket = driver.bucket;
+
+  await bucket.drop().catch(() => null);
+
+  // Create the bucket.
+  const stream = bucket.openUploadStream('gridfstest');
+  const largeBin = Readable.from(bin);
+  await pipeline(largeBin, stream);
+  _id = stream.id;
```

Comment:
None of this needs to happen per-test, right?  Can we put this all in the before hook to same some test execution time?

Reply:
agreed, done